src/components/include/utils/threads/message_loop_thread.h

```diff
@@ -86,6 +86,9 @@ class MessageLoopThread {
   // Process already posted messages and stop thread processing. Thread-safe.
   void Shutdown();
 
+  // Added for utils/test/auto_trace_test.cc
+  size_t GetMessageQueueSize() const;
```

Comment:
Do you use this method somewhere except of unit-tests? 
If no - may be wrap into #ifdef BUILD_TESTS?